porter/v1/datastore.proto

```diff
@@ -22,6 +22,13 @@ message ManagedDatastore {
   oneof kind_values {
     AwsRds aws_rds_kind = 7;
   }
+  // peering describes the peering of this datastore
+  Peering peering = 8;
```

Comment:
Peering is specific to a technology. There could be (and likely will be) clusters connected through vpc endpoints in future. Approving, but would recommend something like `connected_clusters`